modules/log-forwarding-modes.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+
+:_content-type: CONCEPT
+[id="log-forwarding-modes_{context}"]
+= Log forwarding modes
+
+There are two log forwarding modes available: legacy mode, and multi log forwarder mode.
+
+[IMPORTANT]
+====
+Only the Vector collector is supported for use with multi log forwarder mode. The Fluentd collector can only be used with the legacy `instance` log forwarder in the `openshift-logging` namespace.
```

Comment:
> Only the Vector collector is supported for use with multi log forwarder mode.

Does this mean that other than Vector, no other collector can be used with this mode, or Vector cannot be used with legacy mode?

Reply:
> used with the legacy `instance` log forwarder

Is this _legacy mode_?

Reply:
> Does this mean that other than Vector, no other collector can be used with this mode

Yes. The other option is the Fluentd collector (deprecated) and that can't be used with the multi log forwarder mode.

Reply:
Updated for clarity.

Reply:
legacy mode:  The resources  clusterlogging  and ClusterLogForwarder must be deployed under openshift-logging namespace.  and the resource name must be instance.

New mode: 
1. The resources  clusterlogging  and ClusterLogForwarder can be deployed under any namespace.
2. ServiceAccount must be created and specified in ClusterLogForwarder.  
3. ServiceAccount must bind to logging roles(collect-application-logs,collect-audit-logs,                                                         
collect-infrastructure-logs )